src/Sylius/Bundle/ShopBundle/Resources/views/layout.html.twig

```diff
@@ -10,6 +10,7 @@
     <meta content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=no" name="viewport">
 
     {% block metatags %}
+        {{ sonata_block_render_event('sylius.shop.layout.metatags') }}
```

Comment:
This should rather be part of `{{ sylius_template_event('sylius.shop.layout') }}` I suppose /cc @pamil

Reply:
Its true, we replace old `sonata_block_render_event` to `sylius_template_event`, look in this PR : https://github.com/Sylius/Sylius/pull/11126